### ISSUE TITLE
Date Alias Replacement in Body

### ISSUE THREAD

**Issue by bakern** [mpdf/mpdf#1398]
The documentation does state that the `{DATE ...}` alias will only be replaced in headers and footers, however this line from the CHANGELOG for 8.0.x seems to indicate it might be replaced in the body:
```
Allows {nb}, {nbpg}, {PAGENO} and {DATE ...} substitution in body (#172 and #267, @Dasc3er)
```

From what I can tell it's not being replaced in the body, and looking at the code it seems to not be replaced in the `aliasReplace` function because it's already PDF encoded at that point.

I just want to clarify if it should work in the latest version or not, and if not, is there any reason support should not be added?  This issue could also be considered a feature request, and I might be able to submit a pull request if it would help.

Thanks!

**Comment by finwe** [comment]
Yes, replacing date does not work as it is done over already generated PDF data. It must be done already on the WriteHTML call.
